### PR TITLE
[CodePush] Create hermes for iOS

### PR DESCRIPTION
**Pull request by ryanypm**
1: [sourcemap] Support for Hermes on iOS (RN 0.64) #1399
2: Use the original "sourcemap" and "sourcemap.hbc.map" to generate a real sourcemap

When react-native executes the`bundle` command, if `--sourcemap-output` is selected, `index.android.bundle.packager.map` and `index.android.bundle.compiler.map` are used to generate true `index.android.bundle.map`

But `appcenter-cli` didn't do it, and `index.android.bundle.hbc.map` will also be included in the push.

[https://github.com/facebook/react-native/blob/master/react.gradle#L211](https://github.com/facebook/react-native/blob/master/react.gradle#L211)

### CODE REVIEW

**Review by thejoaov — DISMISSED** [review]

**Review by gstvds — APPROVED** [review]

**Review by Krasavinigor — APPROVED** [review]

**Comment by Krasavinigor** [comment]
@ryanypm 
Could you please rename `getHermesEnabled` to `getAndroidHermesEnabled`?

**Review by alexandergoncharov — APPROVED** [review]

File: src/commands/codepush/lib/react-native-utils.ts [hunk 1]
```diff
@@ -423,6 +472,23 @@ export function getHermesEnabled(gradleFile: string): boolean {
     });
 }
 
+export function getiOSHermesEnabled(podFile: string): boolean {
```
Comment (Krasavinigor) [inline src/commands/codepush/lib/react-native-utils.ts:475]: Could you please check this comment https://github.com/microsoft/appcenter-cli/pull/1461#issuecomment-884114207?

**Review by Krasavinigor — APPROVED** [review]

**Review by andreidubov — APPROVED** [review]

**Review by Krasavinigor — APPROVED** [review]

**Review by thejoaov — APPROVED** [review]

**Review by DmitriyKirakosyan — APPROVED** [review]

**Pull request merged**